pkg/pillar/cmd/upgradeconverter/oldglobalconfig.go

```diff
@@ -0,0 +1,400 @@
+package upgradeconverter
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/sirupsen/logrus"
+)
+
+// OldGlobalConfig - Legacy version of global config
+type OldGlobalConfig struct {
```

Comment:
This is error prone, because any update to global.go while we are talking will be missed.

Why not 1) leave the definitions in global.go but rename them to old, and 2) put a comment before each old structure saying that it is retained since it is used by the upgradeconverter